### PR TITLE
dockerized build

### PR DESCRIPTION
**Pull request by gites**
dockerized build for Nautilus-Cloud

### CODE REVIEW

**Comment by bhaskarkishore** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -34,4 +34,12 @@ as described above (take note of the doobie.host config), follow these steps:
...
\ No newline at end of file
+* login-panel: http://localhost:1234/site
+
+#Docker-based build
+
+For Docker base build follow these steps:
+
+    1. Be in the root directory of the repo.
+    2. To build Nautilus-Cloud run `./docker/build/build-images.sh`
+    3. The final container will be named `nautilus-cloud:latest`. The `NC_JAVA_XMS` `NC_JAVA_XMX` env variables can be used to set Java `-Xms` and `-Xmx` parameters. The Nautilus-Cloud config file needs to be mount inside the container in `/nc.conf` location.
```
Comment (ivanopagano) [inline README.md:45]: typo: "needs to be mount-*ed* inside the container"

File: docker/build/build-images.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# build base image
+docker inspect --type=image nautilus-cloud:base-latest &>/dev/null || ./docker/base/build-base-image.sh
+# 2-stage build in docker 
+docker build -t nautilus-cloud -f ./docker/build/dockerfile-build .
```
Comment (ivanopagano) [inline docker/build/build-images.sh:9]: is the path to the `dockerfile-build` correct, relative to where this script will be expected to run from?

Comment (gites) [reply docker/build/build-images.sh:9]: yup. as per instructions you should be in the root directory of the repo.

**Pull request merged**